src/authMappingUtils.js

```diff
@@ -76,3 +78,5 @@ export const userHasMethodOnAnyProject = (method, userAuthMapping = {}) => {
   const hasMethod = actionArrays.some(x => x.some(actionHasMethod));
   return hasMethod;
 };
+
+export const projectIsOpenData = (projectAvail, projectID) => (projectID in projectAvail && projectAvail[projectID] === 'Open');
```

Comment:
not sure why you moved this to `authMappingUtils` since it handles project_ids, not resource paths?

Reply:
oh i was about to put it into https://github.com/uc-cdis/data-portal/blob/ba5c0270de033207cf8eea451360ac7e2713c5b1/src/StudyViewer/StudyDetails.jsx#L83 and don't want to import sth from `CoreMetadataHeader`

but then i realize i have to translate `resource_path` back to `project_id` 🤪

Reply:
should we include open projects in here? ifd so i need to modify this

if not i can move this `projectIsOpenData` back to `CoreMetadataHeader`

Reply:
i don't think we need to, not sure how that would work
since open data should be managed in arborist now instead of sheepdog, this is kind of deprecated anyway

Reply:
cool! i put it back now